kfs-core/src/main/java/edu/arizona/kfs/gl/batch/service/impl/BudgetAdjustmentCashTransferServiceImpl.java

```diff
@@ -585,6 +459,15 @@ protected String getFinancialObjectTypeCode(Integer universityFiscalYear, String
 	    return financialObjectTypeCode;
     }
     
+    public SystemOptions getSystemOptions(Integer universityFiscalYear) {
+    	SystemOptions options = null;
```

Comment:
This and the next line can be combined into one, thus avoiding a "redundant initialization" warning.